src/zenml/config/pipeline_run_configuration.py

```diff
@@ -29,6 +29,7 @@ class PipelineRunConfiguration(
 ):
     """Class for pipeline run configurations."""
 
+    active_stack: Optional[str] = None
```

Comment:
In my opinion, it might be a bit more intuitive to rename this parameter to `stack`.